src/Ocelot/Configuration/File/FileAggregateRoute.cs

```diff
@@ -10,10 +10,7 @@ public class FileAggregateRoute : IRoute
         public string UpstreamHost { get; set; }
         public bool RouteIsCaseSensitive { get; set; }
         public string Aggregator { get; set; }
-
-        // Only supports GET..are you crazy!! POST, PUT WOULD BE CRAZY!! :)
-        public List<string> UpstreamHttpMethod => new() { "Get" };
-
+        public List<string> UpstreamHttpMethod { get; private set; } = new();

```

Comment:
`private set` means you can assign the value by constructor only.
In this case default constructor creates the list.
No much sense to make setter private:
- another properties have public setter
- we can unit test this property